charts/argo-cd/templates/argocd-application-controller/service.yaml

```
@@ -10,6 +10,9 @@ metadata:
   name: {{ template "argo-cd.controller.fullname" . }}
   labels:
     {{- include "argo-cd.labels" (dict "context" . "component" .Values.controller.name "name" .Values.controller.name) | nindent 4 }}
+{{- if .Values.controller.service.labels }}
+{{- toYaml .Values.controller.service.labels | nindent 4 }}
+{{- end }}
```

```suggestion
{{- with .Values.controller.service.labels }}
{{- toYaml . | nindent 4 }}
{{- end }}
```

Can you please use `with` instead of `if`? Same for the other two.  Thank you for the contribution.

---

done